webhook-connector/README.md

```diff
@@ -13,30 +13,32 @@ description: >-
 When an order or shipment is new or has been changed at Tradecloud, we will trigger your webhook.
 
 {% hint style="info" %}
-You can either choose between the **POST** / **PUT** or **GET** webhook.
+You can either choose between the **POST** or **GET** webhook.
 
-**POST or PUT** which will contain the **order event** or **shipment event** in a JSON body.
+**POST** which will contain the **order event** or **shipment event** in a JSON body.
 
 **GET** which will contain the **order ID** or **shipment ID** as path or query parameter.
 
 See [the API manual](https://tradecloud.gitbook.io/api/api/webhook-vs-polling) about pro's and cons of either choice.
 {% endhint %}
 
 {% hint style="warning" %}
-When you **use the event** it will **ONLY** contain the lines **affected** by the event.
+When you **use the order event** it will **ONLY** contain the lines **affected** by the event.
 
-When you **GET the order yourself** you will get **ALL** the lines
+When you **use the shipment event** it will contain **ALL** the lines and **ALL** the dcouments. You can filter new/changed lines and documents based on the `lastChangedAt` fields.
```

Comment:
```suggestion
When you **use the shipment event** it will contain **ALL** the lines and **ALL** the documents. You can filter new/changed lines and documents based on the `lastChangedAt` fields.
```